iron-selectable.html

```diff
@@ -245,6 +245,12 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+
+      if (isSelected) {
+        item.setAttribute('aria-selected', 'true');
+      } else {
+        item.removeAttribute('aria-selected');
+      }
```

Comment:
Yeah, I agree that selectable is abstract and ARIA has a specific meaning wrt UI controls, and ARIA should be implemented in menu.